drafting/conda-template.py

```diff
@@ -0,0 +1,85 @@
+# This script reads in a yaml file (qiime2 recipe)
+# Pulls the needed data from that file
+# And uses that data to write to a secondary yaml file
+# Which will be the end result: a conda recipe
+
+import yaml
+import os
+import sys
+
+# Reading the qiime2 recipe file
+qiime2_recipe = sys.argv[1]
```

Comment:
Can you add a nice error message here if the caller forgets to provide this argument?